tests/baselines/reference/docker/xterm.js.log

```diff
@@ -3,14 +3,37 @@ Standard output:
 
 > xterm@X.X.X build /xtermjs
 > tsc -b ./tsconfig.all.json
+src/browser/Terminal2.test.ts(80,11): error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?
+error TS2688: Cannot find type definition file for '../../../out-test/api/TestUtils'.
+  The file is in the program because:
+    Entry point of type library '../../../out-test/api/TestUtils' specified in compilerOptions
+error TS2688: Cannot find type definition file for '../../../out-test/api/TestUtils'.
```

Comment:
these errors seem worse than before

Reply:
I'm not certain we can know what broke this. None of the projects fixed by #42431 have been building or contributing updated logs for some time now, so there's no way to know *what* caused these breaks. All of these breaks are based on what is in master *now*.

Reply:
The only thing I changed for xterm was the NodeJS version (since xterm started requiring features not present in NodeJS 8). This seems like a problem with their build, not our test (and these user tests often seem to run afoul of that).